apps/openchallenges/challenge-service/src/main/resources/db/submission_types.csv

```diff
@@ -453,48 +453,58 @@
 "452","prediction_file","459","2023-11-11 01:29:20"
 "453","container_image","459","2023-11-11 01:29:20"
 "454","notebook","459","2023-11-11 01:29:20"
-"455","prediction_file","463","2023-11-16 21:57:03"
-"456","other","463","2023-11-16 21:57:03"
-"457","notebook","466","2023-12-06 06:56:06"
-"458","other","466","2023-12-06 06:56:06"
-"459","prediction_file","467","2023-12-06 07:35:00"
-"460","prediction_file","468","2023-12-06 07:52:34"
-"461","prediction_file","469","2023-12-06 08:04:52"
-"462","prediction_file","470","2023-12-06 08:10:49"
-"463","prediction_file","471","2023-12-06 08:19:47"
-"464","prediction_file","472","2023-12-06 08:28:42"
-"465","prediction_file","473","2023-12-06 08:35:31"
-"466","prediction_file","474","2023-12-06 08:45:12"
-"467","prediction_file","475","2023-12-06 08:52:21"
-"468","prediction_file","476","2023-12-06 08:54:24"
-"469","prediction_file","477","2023-12-12 18:31:00"
-"470","prediction_file","478","2023-12-12 21:54:25"
-"471","prediction_file","479","2023-12-12 22:41:48"
-"472","prediction_file","480","2023-12-12 22:42:59"
-"473","container_image","481","2023-12-12 23:22:24"
-"474","other","482","2024-01-09 16:48:56"
-"475","other","483","2024-01-09 16:58:53"
-"476","other","484","2024-01-09 18:36:27"
-"477","other","485","2024-01-09 18:45:21"
-"478","other","486","2024-01-09 18:48:59"
-"479","other","487","2024-01-09 18:51:10"
-"480","notebook","488","2024-01-09 18:59:10"
-"481","notebook","489","2024-01-09 19:02:43"
-"482","notebook","490","2024-01-09 19:08:58"
-"483","other","491","2024-01-11 23:04:48"
-"484","prediction_file","492","2024-01-29 17:48:58"
-"485","container_image","492","2024-01-29 17:48:58"
-"486","other","492","2024-01-29 17:48:58"
-"487","prediction_file","493","2024-01-31 22:49:24"
-"488","prediction_file","494","2024-01-31 22:49:27"
-"489","container_image","495","2024-01-31 22:49:29"
-"490","prediction_file","497","2024-02-06 00:12:34"
-"491","container_image","497","2024-02-06 00:12:34"
-"492","notebook","497","2024-02-06 00:12:34"
-"493","container_image","498","2024-02-12 21:56:27"
-"494","prediction_file","499","2024-02-19 18:20:32"
-"495","mlcube","499","2024-02-19 18:20:32"
-"496","prediction_file","500","2024-03-06 18:57:14"
-"497","container_image","501","2024-03-06 20:12:50"
-"498","prediction_file","502","2024-03-11 22:58:43"
-"499","other","502","2024-03-11 22:58:43"
+"455","prediction_file","460","2023-11-13 22:48:02"
```

Comment:
@gaiaandreoletti Is this correct? The PEGS challenge is listed to have:
- file submission
- container submission
- notebook submission
- MLCube submission
- other submission